conf/httpd.conf.d/httpd.graphite

```diff
@@ -101,10 +101,10 @@ $KeepAliveTimeout = "10";
 $KeepAlive = "On";
 $MaxKeepAliveRequests = "100";
 
-$MaxClients = "56";
-$StartServers = "4";
-$MinSpareServers = "4";
-$MaxSpareServers = "20";
+$MaxClients = "2";
+$StartServers = "1";
+$MinSpareServers = "1";
+$MaxSpareServers = "1";
```

Comment:
Ok, it's a bit late to change that. 
Let's go with your proposal to lower that number of processes and see if we can merge the pools for 5.2.